.github/workflows/pipeline.yml

```diff
@@ -44,5 +44,5 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
       - run: python -m pip install -e .
-      - run: pip install git+https://github.com/matrix-org/matrix-is-tester.git
+      - run: pip install git+git://github.com/H-Shay/matrix-is-tester.git@fix_tests
```

Comment:
This seems to be a change from #362 which we won't want to merge?

Reply:
Yes I just pulled in the tests to show that the code was working-I will change this back now that the fix to IS-tester has been merged.

Reply:
Makes sense, wasn't sure it was done on purpose or not! 👍

Reply:
Seems this still needs to be reverted?

Reply:
I haven't pushed the changes yet-I had the questions about typing below and was just going to push everything once they were sorted out.

Reply:
Ah, OK! I got confused since the conversation was resolved!